internal/provider/resource_gitlab_managed_license.go

```diff
@@ -171,3 +181,20 @@ func projectIdAndLicenseIdFromId(id string) (string, int, error) {
 
 	return projectId, licenseId, nil
 }
+
+func checkDeprecatedValuesForDiff(k, oldValue, newValue string, d *schema.ResourceData) bool {
+	approvedValues := []string{"approved", "allowed"}
+	deniedValues := []string{"blacklisted", "denied"}
+
+	//While we could technically combine these two "if" blocks, this seems more readable
```

Comment:
```suggestion
	// While we could technically combine these two "if" blocks, this seems more readable
```

I love the comment - well, the prio for readability :D